src/OpenFeature.SDK/OpenFeatureClient.cs

```diff
@@ -20,6 +20,13 @@ public sealed class FeatureClient : IFeatureClient
         private readonly FeatureProvider _featureProvider;
         private readonly List<Hook> _hooks = new List<Hook>();
         private readonly ILogger _logger;
+        private readonly EvaluationContext _evaluationContext;
+
+        /// <summary>
+        /// Gets the client <see cref="EvaluationContext"/>
+        /// </summary>
+        /// <returns></returns>
+        public EvaluationContext GetContext() => this._evaluationContext;

```

Comment:
The client now has it's own context, as per spec. It can be supplied in the `.getClient()` method.